lighthouse-core/audits/byte-efficiency/byte-efficiency-audit.js

```diff
@@ -114,11 +114,24 @@ class UnusedBytes extends Audit {
       devtoolsLog,
       settings,
     };
-
     const networkRecords = await NetworkRecords.request(devtoolsLog, context);
+
+    // Requesting load simulator requires non-empty network records.
+    // Timespan is not guaranteed to generate network records, so mark N/A if empty.
```

Comment:
```suggestion
    // Timespans are not guaranteed to have any network activity.
    // There are no bytes to be saved if no bytes were downloaded, so mark N/A if empty.
```

WDYT? just to make super clear it's the logical conclusion not a technical limitation :)